client-registry-jempi/swarm.sh

```diff
@@ -102,14 +91,7 @@ function initialize_package() {
   fi
 
   (
-    log info "Deploy Kafka"
-    docker::deploy_service "${COMPOSE_FILE_PATH}" "docker-compose.kafka.yml"
-    docker::deploy_sanity "${KAFKA_SERVICES[@]}"
-
-    log info "Deploy Kafdrop"
-    docker::deploy_service "${COMPOSE_FILE_PATH}" "docker-compose.kafdrop.yml" "$kafdrop_dev_compose_param"
-    docker::deploy_sanity "jempi-kafdrop"
-
+    log info "Importing JeMPI Kafka topics"
```

Comment:
The env vars in docker compose config importer are still configured to "jempi-kafdrop', kafdrop host should be "kafdrop" right ?, also I think we need to add message-bus-kafka as a dependency to this package